contracts/p1/mixins/Trading.sol

```diff
@@ -17,7 +17,12 @@ import "./RewardableLib.sol";
 ///   changed without breaking <3.0.0 RTokens. The only difference in
 ///   MulticallUpgradeable is the 50 slot storage gap and an empty constructor.
 ///   It should be fine to leave the non-upgradeable Multicall here permanently.
```

Comment:
Quick callout @julianmrodri: Need to verify this is safe. It used to be the standard Multicall but it was complaining if I didn't replace it with the Upgradable import.

Reply:
Won't this break RTokens? If the Upgradeable has an extra 50 storage slot var and this was previously non-upgradeable, then we can't switch over right?

Reply:
Very likely this won't pass the validation but need to run the verifier to confirm tomorrow.

Reply:
Confirmed. Failing to upgrade from 3.0.1 (our latest version) to this on the backing Manager.

<img width="410" alt="image" src="https://github.com/reserve-protocol/protocol/assets/56316686/4ce43316-3401-4a48-bd53-3978ea0da47a">